models/ClimateSensor.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azsamples:iotdd:ClimateSensor;1",
+  "@type": "Interface",
+  "displayName": "Climate Sensor",
+  "description": "Provides functionality to report temperature, humidity, Pressure",
+  "comment": "Requires temperature, hunidity and pressure sensors.",
+  "contents": [
+    {
+      "@type": "Property",
+      "displayName": "Update interval",
+      "description": "Interval of telemetry updates in seconds",
+      "name": "updateInterval",
+      "writable" : true,
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Telemetry",
+        "SemanticType/Temperature"
```

Comment:
Semantic types are not prefixed with "SemanticType/", so this should just be "Temperature".

Reply:
mmhh.. I validated these models with the parser and it did not complain. I will remove the SemanticType for now, but that's something that I'd like to include.

Reply:
I think it's fine to include semantic types. The names of the standard, built-in semantic types do not have a "SemanticType/" prefix, so you can just use "Temperature", "Humidity", etc.

---
In reply to: [417692512](https://github.com/rido-min/pnp-ppr-sample-spec/pull/2#discussion_r417692512) [](ancestors = 417692512)